src/snowflake/snowpark/session.py

```diff
@@ -727,10 +728,11 @@ def add_packages(
         :class:`~snowflake.snowpark.udf.UDFRegistration`. See details of
         `third-party Python packages in Snowflake <https://docs.snowflake.com/en/developer-guide/udf/python/udf-python-packages.html>`_.
 
-        Pure Python packages that are not available in Snowflake will be pip installed locally and made available as an
+        EXPERIMENTAL: Pure Python packages that are not available in Snowflake will be pip installed locally and made available as an
```

Comment:
Better explain how this experimental feature is enabled. For instance, xyz argument is set to True, then the packages will be automatically installed from pip.

Reply:
The feature is enabled when a custom package not present on Anaconda channel is passed to `add_requirements` and `add_packages`. Do you want me to only allow the feature via a specific argument instead?